tutorials/mongoose/find-with-async.md

```diff
@@ -0,0 +1,11 @@
+The Mongoose `find(filter,callback)` function allows you to query for documents with the given key(s)/value(s) and it will return an array of documents that match the given filter. You can use this function with asynchronous calls as follows:
+
+```javascript
+[require:mongoose-find-async find]
+```
+
+If you omit the filter parameter in the `find()` call, it will just return all documents in the form of an array.
```

Comment:
replace the last part with "it will find all documents." and link to the [find all tutorial](https://masteringjs.io/tutorials/mongoose/find-all). Internal linking is important for SEO.